dind/build-cluster.sh

```diff
@@ -29,14 +29,18 @@ echo "Building cluster docker image from ${CLUSTER_DIR}"
 cd ${KUBE_ROOT}
 
 # The outer layer doesn't run node or master components. But tests need kubectl.
-bazel build //build/debs:kubectl
+bazel build //build/debs:kubectl --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64
 
 # Tests are run from the outer layer, so package the test artifacts.
-bazel build //vendor/github.com/onsi/ginkgo/ginkgo:ginkgo
-bazel build //test/e2e:e2e.test
+bazel build //vendor/github.com/onsi/ginkgo/ginkgo:ginkgo --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64
+bazel build //test/e2e:e2e.test --platforms=@io_bazel_rules_go//go/toolchain:linux_amd64
 
 cd -
 
+cd ${TOOL_ROOT}
```

Comment:
I bazel build multiple repos. I ran into some weirdness building when cwd wasn't under the workspace, but that might have been a broken state of my workspace.

Reply:
ah, it should definitely at least but *under* but it can be deeper.

Reply:
Ah, that explains it. I don't want the tool to make any assumptions about it's starting directory, so I changed into the appropriate directories per workspace, since there are multiple.